steps/tryoutapps-shaman/1.0.3/step.yml

```diff
@@ -0,0 +1,73 @@
+title: TryoutApps Shaman
+summary: |
+  Uploads apps to Infinum's TryoutApps
+description: |
+  Uploads apps to Infinum's TryoutApps using shaman deploy script.
+  Only one build at the time can be uploaded.
+website: https://github.com/infinum/TryoutApps-Shaman-Bitrise-Deploy
+source_code_url: https://github.com/infinum/TryoutApps-Shaman-Bitrise-Deploy
+support_url: https://github.com/infinum/TryoutApps-Shaman-Bitrise-Deploy/issues
+published_at: 2019-11-27T07:36:43.159925+01:00
+source:
+  git: https://github.com/infinum/TryoutApps-Shaman-Bitrise-Deploy.git
+  commit: 2523459db62a4070150f918c9fcfafe42957fd6a
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- macos
+- android
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: "Get your user token from https://[TENANT].tryoutapps.com/users/me
+      \n"
+    is_required: true
+    is_sensitive: true
+    title: Token used for auth with TryoutApps
+  shaman_token: null
+- environment_name: null
+  opts:
+    description: |
+      Name defined in shaman.yml
+    is_required: true
+    title: Deploy environment name
+- opts:
+    category: Config
+    description: |
+      By default, this value is set to './.shaman.yml'
+    is_required: true
+    title: Path to the shaman file
+  shaman_config_path: ./.shaman.yml
+- file_path: ${BITRISE_IPA_PATH}
+  opts:
+    category: Config
+    description: |
+      By default, this value is set to "${BITRISE_IPA_PATH}"
```

Comment:
the input value contains this information already, so instead of the current one, describe the requirements for this input if any.